pylabnet/scripts/sweeper/scan_1d.py

```diff
@@ -258,6 +270,35 @@ def save(self, filename=None, directory=None, date_dir=True):
                 date_dir=date_dir
             )
 
+    def fit_config(self, status:bool):
+        """ Configures fitting add-on
+
+        :param status: (bool) whether or not fit button is checked
+        """
+
+        # If box is newly checked, instantiate popup
+        if status:
+            self.fit_popup = FitPopup(ui='fit_popup',
```

Comment:
what happens to the `self.fit_popup` attribute when the box is checked, but then the fit popup window is closed?